runtime/shibuya/src/lib.rs

```diff
@@ -229,6 +230,19 @@ impl_opaque_keys! {
     }
 }
 
+// Adjusting the base extrinsic weight to account for the additional database
+// read introduced by the `tx-pause` pallet during extrinsic filtering.
+//
+// TODO: This hardcoded addition is a temporary fix. Replace it with a proper
+// benchmark in the future.
+pub struct RuntimeAdjustedWeights;
```

Comment:
no need for a struct, just a function
`pub fn extrinsic_base_weight() -> Weight {`